metacat-functional-tests/build.gradle

```diff
@@ -63,18 +67,20 @@ dependencies {
      *******************************/
 
     // Test that metacat-client can be used with jersey 1
-    testCompile(project(":metacat-client")) {
-        exclude group: "javax.ws.rs", module: "javax.ws.rs-api"
-        exclude group: "org.glassfish.jersey.core"
-    }
+    functionalTestShadedCompile project(path: ':metacat-client', configuration: 'shadow')
+
```

Comment:
I don't understand this 'Test that metacat-client can be used with jersey 1'..

Reply:
This was an old comment. Then the platform supported jersey 1. We can remove the comment and dependency in the test.

Reply:
That is really old, from before Metacat moved to Spring.  When we were transitioning to Java 8 I added a test to make sure Metacat worked with both Jersey 1 and 2.  It looks like only the comment remains and can be safely removed.